src/main/target/KAKUTEF4/target.mk

```diff
@@ -0,0 +1,12 @@
+F405_TARGETS    += $(TARGET)
+FEATURES        += VCP ONBOARDFLASH
+
+TARGET_SRC = \
+            drivers/accgyro_spi_mpu9250.c \
+            drivers/barometer_bmp280.c \
+            drivers/barometer_ms5611.c \
+            drivers/compass_ak8963.c \
+            drivers/compass_hmc5883l.c \
+            drivers/light_ws2811strip.c \
+            drivers/light_ws2811strip_stm32f4xx.c
```

Comment:
`drivers/light_ws2811strip.c` and `drivers/light_ws2811strip_stm32f4xx.c` no longer required in `.mk` file